haiku-apps/stopwatch/stopwatch-1.0.0.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="A simple stopwatch application"
+DESCRIPTION="This stopwatch is very simple to use and allows a user to start, stop and reset the clock."
+HOMEPAGE="https://github.com/dcatt-haiku/StopWatch"
+COPYRIGHT="2016 DCatt"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dcatt-haiku/StopWatch/archive/v1.0.0.tar.gz"
+CHECKSUM_SHA256="320f0d7502316e12209c73576cacf488f49abd58a42cc986a84882a54eb9e732"
+SOURCE_DIR="StopWatch-1.0.0"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	StopWatch = $portVersion
+	app:StopWatch = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp -af objects.*/StopWatch $appsDir
```

Comment:
Yep, which would be correct if you didn't use "OBJ_DIR=objects " in your make line. That makes all objects and the executable appear in a "objects" folder instead of one with the used compiler and debug-status in its name, e.g. "objects.debug.x86_gcc2" or something. Then you'd need the wildcard "*" to mask that suffix.   
Found your makefile-engine yet? :)